src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -980,6 +972,17 @@ private void InheritNullableStateOfMember(int targetContainerSlot, int valueCont
             }
         }
 
+        private void SetStateAndTrackForFinally(ref LocalState state, int slot, NullableFlowState newState)
```

Comment:
>SetStateAndTrackForFinally [](start = 21, length = 26)

please add a comment explaining when this method should be used (rather than just doing `state[slot] = newState;`). Thanks